libvisual/tools/lv-tool/display/stdout_sdl_driver.cpp

```diff
@@ -30,6 +30,9 @@
 
 #include <SDL/SDL.h>
 #include <GL/gl.h>
+#ifndef GL_BGR
+#include <GL/glext.h>
+#endif
```

Comment:
Is an unconditional include troublesome? If not, my vote for dropping `#ifndef GL_BGR`.

PS: Would be great if the related commit could mention how `GL_BGR` is defined only in some cases but not in others.

Reply:
I mainly added an `#ifndef` because I only readed it for `GL_BGR` since it seems that for msys2, `GL_BGR` is not defined in `GL/gl.h` as the rest of GL stuff and didn't want to include it if necessary.

I can remove the conditionals and amend the message

Reply:
Maybe?:
```c++
#include <GL/glext.h>  // e.g. for GL_BGR on MinGW
```

Reply:
I used ` // For GL_BGR in systems where it's not defined in GL/gl.h` since I didn't see your message in time

Reply:
Good enough for sure, thanks!